docs/config-file.md

```diff
@@ -12,6 +12,10 @@ This means you can configure Unibeautify via:
 
 The [options](options-for-languages.md) to the configuration file are grouped by the respective language.
 
+## Using the Assistant
+
+Use the Unibeautify assistant, https://assistant.unibeautify.com/, for an interactive setup of your configuration file. Simply select the languages you want, and it will walk you though the options available. At the end simply download or copy to your clipboard your configuration.
```

Comment:
I'm hoping users know to click on links so we can use:

```diff
- Unibeautify assistant, https://assistant.unibeautify.com/
+ [Unibeautify assistant](https://assistant.unibeautify.com/)
```

Reply:
Also I think a direct link to https://assistant.unibeautify.com/#/setup may be a good idea.


```diff
- Unibeautify assistant, https://assistant.unibeautify.com/
+ [Unibeautify Assistant](https://assistant.unibeautify.com/#/setup)
```